test/replcompletions.jl

```diff
@@ -494,6 +493,29 @@ c, r, res = test_scomplete(s)
         ENV["PATH"] = oldpath
     end
 
+    # Make sure completion results are unique in case things are in the env path twice.
+    let
+        # Create /tmp/repl-completion and /tmp/repl-completion-subdir/repl-completion
+        file0 = joinpath(tempdir(), "repl-completion")
+        dir = joinpath(tempdir(), "repl-completion-subdir")
+        mkdir(dir)
+        file1 = joinpath(dir, "repl-completion")
+        touch(file0)
+        touch(file1)
+
+        oldpath = ENV["PATH"]
+        ENV["PATH"] = string(tempdir(), ":", dir)
```

Comment:
cool. a few of the other existing tests that weren't ones you wrote might also be portable, like the `cd $(Pk` one.

source builds on windows take a really long time, so either let that go overnight (the cygwin cross compile instructions are the better maintained option there) or use a binary with changes to the julia source in base made via build_sysimg.jl. If you don't have a linker available then you only get a sys.ji so need to start julia with `-J` to use the new image.